cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -219,7 +221,11 @@ private CristinBookReport extractCristinBookReport() {
     }
 
     private String extractNumberOfPages() {
-        return extractCristinBookReport().getNumberOfPages();
+        String numberOfPages = extractCristinBookReport().getNumberOfPages();
+        if (numberOfPages == null) {
+            throw new MissingFieldsException(ERROR_MISSING_NUMBER_OF_PAGES_IN_BOOK_REPORT);
```

Comment:
I think this is an InvalidCristinBookEntry exception . MissingFieldsException is only for fields that we should fill in but we are not filling in for some reason that we do not know. The InvalidCristinBookEntry exception does not exist yet, it should be a RuntimeException.